definitions/npm/turf-point_v2.x.x/flow_all/turf-point_v2.x.x.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+type $npm$Turf$Point$Point = {
```

Comment:
I redefine this in a few of the definitions because I assume it's not safe to share types across module definitions safely that may or may not get included in the users project